dom0/sd-workstation.top

```diff
@@ -47,6 +45,8 @@ base:
     - sd-mime-handling
   sd-proxy:
     - sd-mime-handling
+  whonix-gateway-17:
+    - sd-whonix-config
```

Comment:
By strict analogy with the Debian templates, this is our cue to clone `whonix-gateway-17` to something like `sd-whonix-gateway-17` and use that as `sd-whonix`'s template, rather than installing our packages directly into `whonix-gateway-17`.  @legoktm, I know you anticipated this in <https://github.com/freedomofpress/securedrop-workstation/pull/1040#issuecomment-2127551263>, but I'm highlighting it explicitly as a change that should be pretty quick to tack on if we decide it's worth being strict about.

Reply:
I'm still trying to reason my way through the implications of cloning the template versus not. My current perspective is that we've already been not cloning it, so we're OK to stick with it. It should be even easier to clean up after ourselves now that everything is packaged.